wp-content/themes/pub/wporg-learn-2020/archive-course.php

```diff
@@ -9,62 +9,101 @@
 
 namespace WordPressdotorg\Theme;
 
-$prev_category = '';
-$category_title = '';
-$category_description = '';
+$all_courses        = array();
+$course_categories  = array();
+$prev_category      = '';
+$begin_new_category = false;
 
 get_header();
 get_template_part( 'template-parts/component', 'breadcrumbs' );
 ?>
 
 <main id="main" class="site-main">
-
 	<section>
-		<div class="row align-middle between section-heading section-heading--with-space gutters">
-			<?php the_archive_title( '<h1 class="section-heading_title h2 col-8">', '</h1>' ); ?>
-			<?php get_template_part( 'template-parts/component', 'archive-search' ); ?>
```

Comment:
Moved to `intro` section, matching Lesson Plan landing page layout

| Lesson Plans | Courses |
|-|-|
| ![Screen Shot 2022-11-30 at 1 38 46 PM](https://user-images.githubusercontent.com/1017872/204679387-e0aefee8-adcd-4d38-b24c-970bac3d4202.jpg) | ![Screen Shot 2022-11-30 at 1 30 08 PM](https://user-images.githubusercontent.com/1017872/204679338-623586e8-aab0-4641-9244-4ba6d42bc417.jpg) |